recipes/openmm-nonbonded-slicing/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+rm -rf build || true
+
+CMAKE_FLAGS="  -DCMAKE_INSTALL_PREFIX=${PREFIX}"
```

Comment:
By using the provided CMAKE_ARGS you should be able to get rid of most of the flags here.